app/javascript/components/pages/EventShow.js

```diff
@@ -52,15 +52,19 @@ class EventShow extends Component {
   getIngredientsMeasurements = (object) => {
     let ingredients = [];
     let measurements = [];
-    for(const [key,value] of Object.entries(object)){
-      if(key.startsWith('strIngredient') && value !== null && value !== ''){
-        ingredients.push(value)
-      }else if(key.startsWith('strMeasure') && value !== null && value !== ''){
-        measurements.push(value)
-    };
-   this.setState({ingredients, measurements});
- };
-};
+    for (const [key, value] of Object.entries(object)) {
+      if (key.startsWith('strIngredient') && value !== null && value !== '') {
+        ingredients.push(value);
+      } else if (
+        key.startsWith('strMeasure') &&
```

Comment:
similar to the comment above, you can create another if statement and check on truthiness
```suggestion
        if (object.strMeasure) {
```